types/pouchdb-core/index.d.ts

```diff
@@ -8,7 +8,8 @@
 
 /// <reference types="debug" />
 /// <reference types="pouchdb-find" />
-/// <reference types="dom" />
+
+interface Blob {}
```

Comment:
Unfortunately, this doesn't make tslint happy.
Maybe, add an inline disable for the `no-empty-interface` rule andadd a comment why the interface is empty

Reply:
Ah, thanks!  I didn't test this properly.

Reply:
Just replaced this with an inline duplication of the `Blob` interface (as was done with `Buffer`).